src/components/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -175,4 +186,8 @@ export default class AddNameTranslation extends Vue {
       }
     }
   }
+  ::v-deep .v-label {
+      color: #495057;
```

Comment:
Please use CSS variable for this color -- see https://github.com/bcgov/business-edit-ui/blob/master/src/assets/styles/theme.scss

Reply:
changed